### ISSUE TITLE
Truncated text

### ISSUE THREAD

**Issue by dalb8** [SubhamTyagi/Last-Launcher#151]
With certain fonts, e.g. italics the end is chopped off or truncated. 
![truncate](https://user-images.githubusercontent.com/726844/164270153-25db3ec7-a811-4a01-b1b7-f92f131e225b.png)

**Comment by Leo-Adventure** [comment]
Hi, may I work on this issue? I have come up some ideas

**Comment by SubhamTyagi** [comment]
> Hi, may I work on this issue? I have come up some ideas

Yes. :  I think only padding and margins setting will fix this issue

**Comment by Leo-Adventure** [comment]
OK, I'm currently work on this~
